File: modulo_1/4.py
n = int(input())

if n % 2 == 0:
    print("es multiplo de 2")
elif n % 3 == 0:
    print("es multiplo de 3")
elif n % 5 == 0:
    print("es multiplo de 5")
elif n % 7 == 0:
    print("es multiplo de 7")
else:
    print("no es multiplo de ninguno de los primeros cuatro primos")

File: modulo_1/cpp/4.cpp
#include <iostream>

using namespace std;

int main() {
    int n;
    cin >> n;

    if (n % 2 == 0) {
        cout << "es multiplo de 2";
    } else if (n % 3 == 0) {
        cout << "es multiplo de 3";
    } else if (n % 5 == 0) {
        cout << "es multiplo de 5";
    } else if (n % 7 == 0) {
        cout << "es multiplo de 7";
    } else {
        cout << "no es multiplo de ninguno de los primeros cuatro primos";
    }

    return 0;
}
